package.json

```diff
@@ -71,6 +71,7 @@
     "opbeat": "^4.14.0",
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
+    "react-dropzone": "^4.0.0",
```

Comment:
This repo is using `yarn`

Make sure you use `yarn add` to add packages, because it seems that the `yarn.lock` file was not updated.